include/zephyr/drivers/video.h

```diff
@@ -819,6 +819,23 @@ void video_closest_frmival_stepwise(const struct video_frmival_stepwise *stepwis
 void video_closest_frmival(const struct device *dev, enum video_endpoint_id ep,
 			   struct video_frmival_enum *match);
 
+/**
+ * @brief Return the link-frequency advertised by a device
+ *
+ * Device exposing a CSI link will should advertise at least one of the following two controls:
+ *   VIDEO_CID_LINK_FREQUENCY
+ *   VIDEO_CID_PIXEL_RATE
+ *
+ * At first the helper will try read the VIDEO_CID_LINK_FREQUENCY and if not available will
+ * approximate the link-frequency from the VIDEO_CID_PIXEL_RATE value, taking into consideration
+ * the bits per pixel of the format and the number of lanes.
```

Comment:
Some help with doc proofreading:
```suggestion
 * Device exposing a CSI link should advertise at least one of the following two controls:
 * - @ref VIDEO_CID_LINK_FREQUENCY
 * - @ref VIDEO_CID_PIXEL_RATE
  *
 * At first the helper will try read the @ref VIDEO_CID_LINK_FREQUENCY and if not available will
 * approximate the link-frequency from the @ref VIDEO_CID_PIXEL_RATE value, taking into consideration
 * the bits per pixel of the format and the number of lanes.
```